flow360/component/simulation/translator/solver_translator.py

```diff
@@ -227,14 +228,28 @@ def inject_isosurface_info(entity: Isosurface):
 
 def inject_probe_info(entity: EntityList):
     """inject entity info"""
+    if isinstance(entity.stored_entities[0], PointArray):
+        return {
+            "start": [item.start.value.tolist() for item in entity.stored_entities],
+            "end": [item.end.value.tolist() for item in entity.stored_entities],
+            "numberOfPoints": [item.number_of_points for item in entity.stored_entities],
+            "type": "lineProbe",
+        }
     return {
         "monitorLocations": [item.location.value.tolist() for item in entity.stored_entities],
         "type": "probe",
     }
 
 
-def inject_surface_robe_info(entity: EntityList):
+def inject_surface_probe_info(entity: EntityList):
     """inject entity info"""
+    if isinstance(entity.stored_entities[0], PointArray):
```

Comment:
Can we add validation that there is only 1 type of entity in entity list?
BTW can you remind me why we decided this way? Why not append isolated points to the line?

Reply:
Yes, the validation is added